spec.html

```diff
@@ -6572,6 +6572,12 @@ <h1>
         1. If IsConstructor(_S_) is *true*, return _S_.
         1. Throw a *TypeError* exception.
       </emu-alg>
+      <emu-note>
+        <p>Species constructor is only available for Array, RegExp and Promise built-in classes. It
+        will not be provided to any other built-in classes. Species are an expensive and complex
+        operation that did not end up being used by the community, resulting in the feature being
+        deprecated.</p>
```

Comment:
Editors discussed this and we think that capturing "we aren't going to do this anymore" probably doesn't belong in the spec itself. Instead, [how-we-work/normative-conventions.md](https://github.com/tc39/how-we-work/blob/main/normative-conventions.md) would be a good place for capturing this for the committee's benefit, and MDN for the community's.

Reply:
Should this note be removed entirely or should the comment about species constructor availability stay?